in_toto/resolver/__init__.py

```diff
@@ -0,0 +1,41 @@
+# Copyright New York University and the in-toto contributors
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+<Program Name>
+  __init__.py
+
+<Author>
+  Alan Chung Ma <achungma@purdue.edu>
+
+<Started>
+  February 1, 2023
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Build resolver scheme dictionary and set default resolver.
+
+"""
+
+from in_toto.resolver.file_resolver import FileResolver
+from in_toto.resolver.resolver import (
+  DEFAULT_SCHEME,
+  RESOLVER_FOR_URI_SCHEME,
+  Resolver,
+  get_scheme,
+)
+
+RESOLVER_FOR_URI_SCHEME.update(
+  {
+    DEFAULT_SCHEME: FileResolver,
+    FileResolver.SCHEME: FileResolver,
+  }
+)
+
+def set_params(**kwargs):
```

Comment:
May I ask, why you favor this approach over [my suggestion](https://github.com/in-toto/in-toto/pull/536#discussion_r1137239643)? 

`set_params` hides from the caller, which resolver they are configuring, but I don't see the advantage over being explicit.

Reply:
Was discussed in a meeting earlier today.

I was trying to solve a problem that didn't exist and it wouldn't have worked. Will have the configs be set explicitly.